src/Header/Nav/NavLinks/NavLinks.test.jsx

```diff
@@ -59,6 +59,15 @@ describe("NavLinks", () => {
 		expect(link.props()["aria-current"]).toBe(true);
 	});
 
+	it.only("should add render an abbreviation title element with an id that matches the current service id", () => {
+		const wrapper = shallow(
+			<NavLinks {...defaultProps} currentService="link2" />
+		);
+		const link = wrapper.find("[id='link2']");
+		expect(link.props()["aria-label"]).toBe("Abbreviation title");
+		expect(link.props()["id"]).toEqual("link2");
```

Comment:
If we're removing the id then I think we can remove this test, or at least the assertion